FAQ.md

```diff
@@ -16,7 +16,11 @@ WARNING: Please consider reporting this to the maintainers of org.apache.ibatis.
 WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
 WARNING: All illegal access operations will be denied in a future release
 ```
-This warning is shown when using JDK 9 and later. It will need to be fixed by MyBatis.
+This warning is shown when using JDK 9 and later.<br>
```

Comment:
please replace `<br>` with two spaces to force a new line, see also https://gist.github.com/shaunlebron/746476e6e7a4d698b373

Reply:
done.